examples/misc/lib/articles/io/io_http_server_file_test.dart

```diff
@@ -0,0 +1,44 @@
+// #docregion
+import 'dart:async' show Future;
+import 'dart:io';
+
+Future<void> runServer(String basePath) async {
+  HttpServer server = await HttpServer.bind('127.0.0.1', 8082);
```

Comment:
Yep: https://www.dartlang.org/guides/language/effective-dart/design#avoid-type-annotating-initialized-local-variables

So it looks like I should use final or var here. I think we decided to err on the side of using `var`, since the performance implications of `final` aren't clear yet (varies by platform).